jbpm-designer-client/src/main/resources/org/jbpm/designer/public/stencilsets/bpmn2.0jbpm/stencildata/bpmn2.0jbpm.orig

```diff
@@ -238,19 +238,32 @@
 			"name":"boundaryevent",
 			"properties":[
 				{
-					"id":"boundarycancelactivity",
-					"type":"Boolean",
-					"title":"CancelActivity",
-					"value":false,
-					"description":"Denotes whether the activity should be cancelled or not, i.e., whether the boundary catch event acts as an error or an escalation. If the activity is not cancelled, multiple instances of that handler can run concurrently.",
-					"description_ja":"アクティビティをキャンセルするかどうか、つまり、Boundaryキャッチイベントがエラーになるかエスカレーションになるかを示します。 アクティビティがキャンセルされない場合、そのハンドラの複数のインスタンスを同時に実行できます。",
-					"readonly":false,
-					"optional":true,
-					"inverseBoolean":false,
-					"refToView": [
-                                                "frame",
-                                                "frame2"
-                                        ]                                               
+						"id":"boundarycancelactivity",
+						"type":"Choice",
+						"title":"Cancel Activity",
+						"value":"false",
+						"description":"Denotes whether the activity should be cancelled or not, i.e., whether the boundary catch event acts as an error or an escalation. If the activity is not cancelled, multiple instances of that handler can run concurrently.",
+						"description_ja":"アクティビティをキャンセルするかどうか、つまり、Boundaryキャッチイベントがエラーになるかエスカレーションになるかを示します。
+アクティビティがキャンセルされない場合、そのハンドラの複数のインスタンスを同時に実行できます。",
```

Comment:
@tsurdilo Somehow a newline has been inserted into the "description_ja" property (I guess this may have happened when you pushed to github).  This breaks the json, so I wasn't able to test the PR as it stands.